conf/openmetadata.yaml

```diff
@@ -132,6 +132,7 @@ authorizerConfiguration:
   adminPrincipals: ${AUTHORIZER_ADMIN_PRINCIPALS:-[admin]}
   botPrincipals: ${AUTHORIZER_INGESTION_PRINCIPALS:-[ingestion-bot]}
   principalDomain: ${AUTHORIZER_PRINCIPAL_DOMAIN:-""}
```

Comment:
we cannot have a empty string here. Lets put open-metadata.org and if its empty string we should validate it and throw an error on server start
if this is empty admins/bots will be created without email and this will cause an error during sign-in